apps/chrome-devtools/tsconfig.build.components.json

```diff
@@ -1,8 +1,13 @@
 {
   "extends": "./tsconfig.build",
   "compilerOptions": {
-    "composite": true
+    "composite": false
```

Comment:
why is it not working?

Reply:
This tsconfig is used to compile with esbuild and should not be a composite 
I don't understand what has changed and raised an issue
but setting composite to false (which is the expected value) solve the issue